Non declared dependency: lodash

I got this error after installing prettier-java with `npm install -g prettier-plugin-java`:
```
$ prettier --version
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module 'lodash'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (C:\Users\aurelien.mino\AppData\Roaming\npm\node_modules\prettier-plugin-java\node_modules\java-parser\src\comments.js:2:11)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)

```

lodash is indeed used a lot both in java-parser and prettier-plugin-java, but doesn't not appear in declared dependencies.

--- a/packages/java-parser/package.json
+++ b/packages/java-parser/package.json
@@ -6,7 +6,8 @@
   "repository": "https://github.com/jhipster/prettier-java/tree/master/packages/java-parser",
   "license": "Apache-2.0",
   "dependencies": {
-    "chevrotain": "4.7.0"
+    "chevrotain": "4.7.0",
+    "lodash": "4.17.15"
   },
   "scripts": {
     "test": "npm-run-all clone-samples test:execute",

--- a/packages/prettier-plugin-java/package.json
+++ b/packages/prettier-plugin-java/package.json
@@ -7,7 +7,8 @@
   "license": "Apache-2.0",
   "dependencies": {
     "java-parser": "^0.3.2",
-    "prettier": "1.18.2"
+    "prettier": "1.18.2",
+    "lodash": "4.17.15"
   },
   "scripts": {
     "test": "npm-run-all test:unit test:e2e-core",

